src/modules/idyntree-yarp-visualizer/README.md

```diff
@@ -0,0 +1,31 @@
+# idyntree-yarp-visualizer
+
+This module allows visualing a robot starting from its URDF. 
```

Comment:
visualing -> visualizing

Reply:
Fixed in https://github.com/robotology/idyntree-yarp-tools/pull/7/commits/7c1d30ce3bed4ab628a541e1ddbe6c43769194a6